package/objects/TV_Serie__c.object

```diff
@@ -0,0 +1,367 @@
+                <value>
+                    <fullName>History</fullName>
+                    <default>false</default>
+                    <label>History</label>
+                </value>
+                <value>
+                    <fullName>Documentary</fullName>
+                    <default>false</default>
+                    <label>Documentary</label>
+                </value>
+                <value>
+                    <fullName>War</fullName>
+                    <default>false</default>
+                    <label>War</label>
+                </value>
+                <value>
+                    <fullName>Sport</fullName>
+                    <default>false</default>
+                    <label>Sport</label>
+                </value>
+                <value>
+                    <fullName>Thriller</fullName>
+                    <default>false</default>
+                    <label>Thriller</label>
+                </value>
+                <value>
+                    <fullName>Mystery</fullName>
+                    <default>false</default>
+                    <label>Mystery</label>
+                </value>
+                <value>
+                    <fullName>Fantasy</fullName>
+                    <default>false</default>
+                    <label>Fantasy</label>
+                </value>
+                <value>
+                    <fullName>Adventure</fullName>
+                    <default>false</default>
+                    <label>Adventure</label>
+                </value>
+                <value>
+                    <fullName>Crime</fullName>
+                    <default>false</default>
+                    <label>Crime</label>
+                </value>
+                <value>
+                    <fullName>Romance</fullName>
+                    <default>false</default>
+                    <label>Romance</label>
+                </value>
+                <value>
+                    <fullName>Biography</fullName>
+                    <default>false</default>
+                    <label>Biography</label>
+                </value>
+                <value>
+                    <fullName>Musical</fullName>
+                    <default>false</default>
+                    <label>Musical</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+        <visibleLines>4</visibleLines>
+    </fields>
+    <fields>
+        <fullName>Logo__c</fullName>
+        <externalId>false</externalId>
+        <formula>IMAGE(Logo_url__c , &quot;logo&quot;, 100,100)</formula>
+        <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
+        <label>Logo</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Logo_url__c</fullName>
+        <externalId>false</externalId>
+        <label>Logo_url</label>
+        <required>true</required>
```

Comment:
I know that it helps with formula in Logo__c, but required for Logo url means that when you want to save tv serie record you have to give some value here (even in tests). Change formula in Logo__c field into: 
IF (NOT(ISBLANK( Logo_URL__c )),IMAGE( Logo_URL__c , 'logo';, 285, 200),'No Logo Given')
Then Logo_url__c doesn't have to be required :)